### DISCUSSION TITLE
Introduction and Request for Guidance on Contributing to mesa

### DISCUSSION THREAD

**Discussion by sumana-2705** [mesa/mesa#2569] [Q&A]
Hello👋
My name is Sumana Sree. I'm a Master's student at IIT (BHU), specializing in Machine Learning and AI. I enjoy contributing to open-source projects and have recently completed Hacktoberfest with more than 4 merged PRs. I am new to this organisation. I want to contribute to **projectmesa**, I have experience in OpenCV, Deep Learning and NLP. Can anyone help to find the issues related to these topics in the repositories.

**Comment by EwoutH** [comment]
Welcome to Mesa! We wrote something specially for you:
- https://github.com/projectmesa/mesa/blob/main/CONTRIBUTING.md#i-have-no-idea-where-to-start

If you have any ideas or questions let us know!
